app/react/src/client/preview/index.ts

```diff
@@ -27,6 +27,7 @@ export const storiesOf: ClientApi['storiesOf'] = (kind, m) => {
 
 export const configure: ClientApi['configure'] = (...args) => api.configure(...args, framework);
 export const addDecorator: ClientApi['addDecorator'] = api.clientApi.addDecorator;
+export type StoryDecorator = Parameters<typeof addDecorator>[0];
```

Comment:
TIL

Reply:
I personally think the name StoryDecorator is a bit verbose, either DecoratorFn or DecoratorFunction would be a bit better IMHO.

This IS what it's called in the addons package btw.

Reply:
I am open to change the name. I was just copying what the definitely typed package named it to cause the least friction. But will change it to anything you want as it's pretty easy to replace it in my stories.